client-js/app/elements/labrad-registry.ts

```diff
@@ -223,7 +223,15 @@ export class LabradRegistry extends polymer.Base {
       return;
     }
 
-    const key = event.detail.combo;
+    const key = event.detail.combo,
+          keyboardEvent = event.detail.keyboardEvent,
+          hasModifiers = (keyboardEvent.shiftKey || keyboardEvent.ctrlKey ||
```

Comment:
I agree; very non-intuitive. I guess we could go complain on the iron-a11y-keys element (or submit a fix...).